README.adoc

```diff
@@ -1,63 +1,73 @@
-:page-tags: ['Java EE']
+:guide-author: Open Liberty
+:page-tags: ['Java EE', 'sessions', 'session persistence', 'sessionCache']
 :page-related-guides: ['rest-intro', 'microprofile-openapi']
 :page-permalink: /guides/{projectid}
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
-:page-seo-title: Caching and persisting session data
+:page-seo-title: Caching HTTP session data in microservices
 :page-seo-description: Learn how to create and cache session data that persists between servers.
 :imagesdir: assets/
 = Caching HTTP sessions in microservices
 
-Learn how to create and work with session data. Expand on this knowledge by using
-the sessionCache feature to persist session data between servers.
+Learn how to create and work with session data that persists between servers using the `sessionCache` feature in Open Liberty.
 
 == What you'll learn
-You'll learn how to set and get session data, about session and session persistence
-theory, and create a microservice that will persist a user's shopping cart data.
-You will run this microservice on two servers accessing the same session cache.
-This guide uses the MicroProfile OpenAPI as a UI for interacting with the servers.
+You'll learn about session and session persistence theory. You'll also learn how to set
+and get session data in microservices. For security reasons, using
+`sessions` is preferred over using `cookies` for sensitive data. Sessions hide data
+from users. Cookies store data on the client's computer, and can be manipulated by a
+savvy user to make fake requests to your site.
+
+The microservice that you'll be working with is a shopping cart web service. You'll
+learn how to persist a user's shopping cart data between servers using
+the `sessionCache` feature in Open Liberty.
+The `sessionCache` feature persists HTTP sessions using Java Caching (JCache).
+You can have high performance HTTP session persistence without the use of a
+relational database. Furthermore, failover of HTTP sessions can be achieved by
+configuring multiple servers to persist data to the
+same location.
 
+=== What is a session?
 A `session` is a way to store information to be used across multiple pages.
 When you work with a local application (such as a word processor), you open it,
 make changes, and then you close it. The computer knows that you opened the app,
-made changes, and closed the app. On the internet, however, the web server does
-not know who you are, or what you do, because it is just processing stateless HTTP
-requests. Session variables store user information like usernames, items in a cart, etc.
-By default, session variables will timeout after 30 minutes of being unused.
+made changes, and closed the app. On the internet, however, the web server doesn't
+know who you are, or what you do, because it's just processing stateless HTTP
+requests. Session variables store user information like user names, items in a cart.
```

Comment:
Since there are only two examples given, should it be "like user names or items in a cart"?

Reply:
Fixed